instrumentation/mongo/mongo-3.1/library/src/test/groovy/io/opentelemetry/instrumentation/mongo/v3_1/MongoDbAttributesExtractorTest.groovy

```diff
@@ -5,15 +5,15 @@
 
 package io.opentelemetry.instrumentation.mongo.v3_1
 
-import static io.opentelemetry.instrumentation.mongo.v3_1.MongoTracingBuilder.DEFAULT_MAX_NORMALIZED_QUERY_LENGTH
-import static java.util.Arrays.asList
-
 import org.bson.BsonArray
 import org.bson.BsonDocument
 import org.bson.BsonInt32
 import org.bson.BsonString
 import spock.lang.Specification
 
+import static io.opentelemetry.instrumentation.mongo.v3_1.MongoTracingBuilder.DEFAULT_MAX_NORMALIZED_QUERY_LENGTH
```

Comment:
Is it really default IntelliJ's import order for groovy? Quite weird :)

Reply:
ya, until we can enforce groovy import order via tooling, seems best to stick with Intellij default